test/csuite/wt10461_skip_list_stress/main.c

```diff
@@ -0,0 +1,263 @@
+/*-
```

Comment:
This test catches the error from WT-10461, but it's rare unless we bump up SKIP_PROBABILITY in misc.h.
@korteland and I will co-ordinate so this test can use `denser_skiplist` which will be added in WT-10525.